dashboard/config/libraries/NativeSpriteLab.interpreted.js

```diff
@@ -0,0 +1,171 @@
+function draw() {
+  executeDrawLoopAndCallbacks();
+}
+
+/* Legacy code only. Do not add any new code below here */
+function clickedOn(spriteId, callback) {
+  spriteClicked('when', spriteId, callback);
+}
+
+function draggable() {
+  return {func: draggableFunc(), name: 'draggable'};
+}
+
+function pointInDirection(spriteId,direction) {
+  if (direction== "North") {
+    setProp(spriteId, 'rotation', 360);
+  }
+  else if (direction== "East") {
+    setProp(spriteId, 'rotation', 90);
+  }
+  else if (direction=="South") {
+    setProp(spriteId, 'rotation', 180);
+  }
+  else if (direction=="West") {
+    setProp(spriteId, 'rotation', 270);
+  }
+  else {
+    console.error("pointInDirection: invalid direction provided");
```

Comment:
Might be worth adding a return here in case anyone adds something else to the end.